### PR TITLE
First commit

### PR DESCRIPTION
**Pull request by manuelleungchen**
Adding index file

Adding inline CSS to the index file

@mattmckerroll Please review this code.

### CODE REVIEW

**Comment by HustinKava** [comment]
looks good

File: index-leung-chen.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en-us">
+    <head>
+
```
Comment (mattmckerroll) [inline index-leung-chen.html:4]: **_this is pretty neat!_**

Comment (HustinKava) [reply index-leung-chen.html:4]: wow

File: index-leung-chen.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        <!-- Page Title -->
+        <title>Manuel Leung</title>
+
+    </head>
+    <body>
+        <header>
+            <h1 style="color: #493db3;">Manuel Leung Chen</h1>
+        </header>
+
+        <h2 style="color: rgb(40, 104, 104);">Current Goal</h2>
```
Comment (mattmckerroll) [inline index-leung-chen.html:14]: _inline styles???_

**Review by mattmckerroll — APPROVED** [review]
read the comments!

**Review by HustinKava — APPROVED** [review]

**Pull request merged**